packages/affiliates/payouts/2021-02-08_2021-02-15.json

```diff
@@ -0,0 +1,80 @@
+{
+  "config": {
+    "startTime": 1612220400000,
+    "endTime": 1612825200000,
+    "empWhitelist": [
+      ["0xaB3Aa2768Ba6c5876B2552a6F9b70E54aa256175", "0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9"],
+      ["0x3a93E863cb3adc5910E6cea4d51f132E8666654F", "0x53911776641d6dF38B88b9eF27f920c617E3Cb5e"],
+      ["0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
+      ["0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
+      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8"],
+      ["0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037", "0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87"],
+      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "0x531B290230b1809d0AE29Ea834EdF96748F49Bf9"],
+      ["0x2862A798B3DeFc1C24b9c0d241BEaF044C45E585", "0xa1eD088d0789e7baBa18Ed4d87baca65376850d8"],
+      ["0xd81028a6fbAAaf604316F330b20D24bFbFd14478", "0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc"],
+      ["0x7c4090170aeADD54B1a0DbAC2C8D08719220A435", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
+      ["0xaD3cceebeFfCdC3576dE56811d0A6D164BF9A5A1", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
+      ["0xC843538d70ee5d28C5A80A75bb94C28925bB1cf2", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E"],
+      ["0xeFA41F506EAA5c24666d4eE40888bA18FA60a1c7", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E"]
+    ],
+    "fallbackPrices": [
+      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "1"],
+      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "1"],
+      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "1"],
+      ["0x2862a798b3defc1c24b9c0d241beaf044c45e585", "1"],
+      ["0xd81028a6fbaaaf604316f330b20d24bfbfd14478", "1"]
+    ],
+    "totalRewards": "50000",
+    "network": 1
+  },
+  "startBlock": {
```

Comment:
Timestamps and block numbers look one week outdated

Reply:
thanks sean, had a mixup with the scripts. reran it with correct dates.